src/Dev/Validation/DatabaseAdminExtension.php

```diff
@@ -9,7 +9,6 @@
 /**
  * Hook up static validation to the deb/build process
  *
- * @method DatabaseAdmin getOwner()
```

Comment:
This is fine for now as the generics work will likely change the way we were doing this anyway.